README.md

```diff
@@ -1,2 +1,36 @@
 # Ogooreck
+
 Sneaky Test library
```

Comment:
More description here about what the library does would be useful

Reply:
Added more in recent commit 🙂